### PR TITLE
fix(test): snapshot renaming

### PR DESCRIPTION
**Pull request by y3rsh**
# Cleanup snapshots 

- Given the new protocol organization and naming
  - delete all the snapshots of the old naming `find -type f ! \( -name '*_S_*' -o -name '*_X_*' \) -exec rm {} +`
  - `make build-opentrons-analysis`  (default on edge)
  - ` make snapshot-test-update`
  - find old snapshots and delete

### CODE REVIEW

File: app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[12a2a22254][Flex_S_v2_15_P1000M_P50M_GRIP_HS_MB_TC_TM_IlluminaDNAEnrichment].json
Comment (y3rsh) [inline app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[12a2a22254][Flex_S_v2_15_P1000M_P50M_GRIP_HS_MB_TC_TM_IlluminaDNAEnrichment].json:1]: ## 👀 on this

seems a big change

File: app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[1960aa7a4c][Flex_S_v2_15_P1000M_P50M_GRIP_HS_MB_TC_TM_IlluminaDNAEnrichmentv4].json
Comment (y3rsh) [inline app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[1960aa7a4c][Flex_S_v2_15_P1000M_P50M_GRIP_HS_MB_TC_TM_IlluminaDNAEnrichmentv4].json:1]: ## 👀 on this but same as others

File: app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[d8cb88b3b2][Flex_S_v2_16_P1000_96_TC_PartialTipPickupSingle].json
Comment (y3rsh) [inline app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[d8cb88b3b2][Flex_S_v2_16_P1000_96_TC_PartialTipPickupSingle].json:1]: ## 👀 at this

File: app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[12a2a22254][Flex_S_v2_15_P1000M_P50M_GRIP_HS_MB_TC_TM_IlluminaDNAEnrichment].json
Comment (sanni-t) [reply app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[12a2a22254][Flex_S_v2_15_P1000M_P50M_GRIP_HS_MB_TC_TM_IlluminaDNAEnrichment].json:1]: This is an expected change following #15002. We now use the latest pipette version's flow rates as the default rates during protocol analysis/ simulation. 
For this particular protocol, the P50 pipette's default flow rate has changed from 8 to 35 uL/sec, default dispense changed from 8 to 57 uL/sec, etc

File: app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[d8cb88b3b2][Flex_S_v2_16_P1000_96_TC_PartialTipPickupSingle].json
Comment (SyntaxColoring) [reply app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[d8cb88b3b2][Flex_S_v2_16_P1000_96_TC_PartialTipPickupSingle].json:1]: @y3rsh You resolved this thread—this is expected? Out of curiosity, do you know why?

**Review by SyntaxColoring — APPROVED** [review]
This all looks reasonable to me, though it's difficult to be exhaustive because of the assorted semantic changes that are happening concurrently with the renames.

This is reaffirming for me that we should run these snapshot tests on each PR, to spread the work of verifying them and keep the diffs tied to the underlying changes that caused them, but I'm curious if you have any alternative solutions.

**Comment by y3rsh** [comment]
> This all looks reasonable to me, though it's difficult to be exhaustive because of the assorted semantic changes that are happening concurrently with the renames.
> 
> This is reaffirming for me that we should run these snapshot tests on each PR, to spread the work of verifying them and keep the diffs tied to the underlying changes that caused them, but I'm curious if you have any alternative solutions.

Yes.  With the better error handling and organization in place, part 3 or 4 of my current effort is to get these running on PR.

**Pull request merged**